code/game/objects/structures/vehicles/lightcycle.dm

```diff
@@ -0,0 +1,252 @@
+/obj/item/key/lightcycle
+	name = "light rod"
+	desc = "A strange-looking glowing rod."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "lightcycle_keys_inactive"
+	w_class = W_CLASS_TINY
+	var/cycle_active = FALSE
+	var/obj/structure/bed/chair/vehicle/lightcycle/summoned_cycle = null
```

Comment:
Also, the vehicle code for both keys and vehicles themselves track eachother already, any reason you can't use the key's paired vehicle variable and vice versa?